### ISSUE TITLE
Neglect case sensitivity

### ISSUE THREAD

**Issue by 0xCoto** [HeRTA/FRBSTATS#17]
While `/FRBSTATS` is recognized, `/frbstats` is not (case-sensitive URL). This should be fixed in order to help accessibility.
